scripts/check_formatting.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+set -eu
+
+if [ -d .github ]; then
+    : pass
```

Comment:
That's a good idea. Maybe we should write a loop, to make this work from inside nested folders as well?

Reply:
I dunno. I don’t think there is a strong case for running it from within a nested directory and in general I usually try to avoid looping through all parent directories in situations like this. It sometimes may lead to usability issues (e.g. if /home is autofs) but we also may end up in someone’s home directory since it’s not inconceivable they have `~/.github` directory.

Reply:
Yes, that's a good point.

If we only use this script to check formatting in the current directory.  It may be a good idea to check only the parent folder of `scripts/check_formatting.sh`. This way we will be able to call the script from any location.

Just my thoughts. Current design is fine as well.